save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/TopBar.kt

```diff
@@ -123,15 +149,17 @@ fun topBar() = FC<TopBarProps> { props ->
                                     }
                                 } else {
                                     // small hack to redirect from history/execution to history
-                                    val resultingLink = currentLink.removeSuffix("/execution")
+                                    val resultingLink = currentPath.removeSuffix("/execution")
                                     a {
                                         href = resultingLink
                                         className = ClassName("text-light")
                                         +pathPart
                                     }
                                 }
                             }
-                            currentLink
+                            if (insideTab != null && index == 0) {
```

Comment:
Isn't this logic duplicated in this function?

Reply:
Yes, First I change the links on the prefixes "organization", "project" etc
Then I delete them from the address and compose the addresses again